torchci/lib/fetchHud.ts

```diff
@@ -97,7 +125,7 @@ export default async function fetchHud(params: HudParams): Promise<{
   );
 
   const commitsBySha = _.keyBy(commits, "sha");
-  let results = hudQuery.results;
+  let results = hudQuery;
```

Comment:
Nit: This looks a bit out of place as you just assign `const hudQuery = response as any[]` above without doing anything with it and then assign it to another variable here.  It seems that having one variable like `const hudResults = response as any[]` is sufficient?